### ISSUE TITLE
Icons can not found from img-folder 

### ISSUE THREAD

**Issue by blackcrack** [mwalbeck/nextcloud-breeze-dark#142]
**Describe the bug**
can't fint the images (here svg images)
**To Reproduce**
1. softlinked in via ls -s the 
2. copy the img-folder unter css
3. add the both files with correkt path to the whole icons
4. include in server the import
5. change the versionsnumber
6. See [strike]error[/strike] Nothing - no icons but the path is right... moment.. chmod ?
7. no, iss all "d" is 751 ! this should work under apache:apache
(Openmandriva 3.0 the whole themes has works under ./themes)

**Expected behavior**
change the Icons to the svg-icons who i have in use and setting up

**Screenshots**
![Screen 00004](https://user-images.githubusercontent.com/7925717/82886162-6d63b700-9f46-11ea-9ce6-12bcaf38e5c5.jpg)  
![Screen 00005](https://user-images.githubusercontent.com/7925717/82885443-7dc76200-9f45-11ea-8644-77a2dac69a92.jpg)

- Windows-Workstation
- Openmandriva 3.X + Samba
- internal softlinked into the Samba-share
- working from Workstation with Samba and SmarttY (someting like Putty/Kitty, but more comfortable)
- copy all files via apache:apache  and the folders have 751 + Files 644

css are read, but img/*.svg not

**Setup (please complete the following information):**
 - Nextcloud version: [17.0.6]
 - Theme version [ 17.0.11]
 - Browser and browser version [Firefox 76.0.1]

**Additional context**
normally should the icons found..... imho 

in /theme/neXtrealTime works:

![Screen 00006](https://user-images.githubusercontent.com/7925717/82887509-676ed580-9f48-11ea-8010-5240503e9dcb.jpg)

**Comment by mwalbeck** [comment]
You're using Nextcloud in a subfolder and you are using apache without beautifying the URL, so you most likely need to add /nextcloud/index.php/ in front of the icon path so it becomes
```
/nextcloud/index.php/apps/nextrealtime/css/img/actions/user.svg
```

**Comment by blackcrack** [comment]
if i add  ../ become i a path like :  
/nextcloud/index.php/apps/nextrealtime/css/../img/actions/user.svg  
so can i add only the path behind or inside of the app-theme folder to the image .  
so have i not the path up to the img-folder in the hand  
because the part "/nextcloud/index.php/apps/nextrealtime/css/" if given from nextcloud

**Comment by blackcrack** [comment]
i have set it up now like this.. in any folder a _icons.scss
server.scss
```
// General and Nextcloud Core Themed images
@import 'server/icons';
@import 'server/actions/icons';
@import 'server/categorys/icons';
@import 'server/filetypes/icons';
@import 'server/places/icons';
```
looks like :  
server/actions/icons
```

.icon-add {
	background-image: url('add.svg');
}
.icon-category-installed {
	background-image: url('user.svg');
}

.icon-category-enabled {
	background-image: url('checkmark.svg');
}

.icon-category-disabled {
	background-image: url('close.svg');
}
~~~~~~~~
~~~~~~
~~~
```


and we should also looking how we can customizing 
- [ ] var(--logo)
- [ ] var(--header-logo)
- [ ] var(--favico-logo) 
?

**Comment by blackcrack** [comment]
nope, works also not :\

![Screen 00007](https://user-images.githubusercontent.com/7925717/82984075-b15fc600-9ff1-11ea-98cd-ffb14b0d80f7.jpg)

so..
either it's because of NC17 and NC18 versioning

or it's a real bug who does not display the pictures..

**Comment by blackcrack** [comment]
![Screen 00008](https://user-images.githubusercontent.com/7925717/82984730-03551b80-9ff3-11ea-9f7a-625ec066780e.jpg)

**Comment by blackcrack** [comment]
hmm...


because i have wrong imported, it is the folder by side of server.scss and i must 
remove server, because it is in img -folder ... 
i have done now all right, but now is the icon away like in the case before..
see ..

![Screen 00010](https://user-images.githubusercontent.com/7925717/82985508-6c895e80-9ff4-11ea-9606-683613a797c8.jpg)

**Comment by blackcrack** [comment]
![Screen 00011](https://user-images.githubusercontent.com/7925717/82986318-ec63f880-9ff5-11ea-91d2-2262982fac03.jpg)  
------ohne pointer-info----------------------------------------------------------------------------
![Screen 00012](https://user-images.githubusercontent.com/7925717/82986636-7dd36a80-9ff6-11ea-8987-7c95b47cda1f.jpg)
------mit pointer-info----------------------------------------------------------------------------
![Screen 00013](https://user-images.githubusercontent.com/7925717/82986646-8330b500-9ff6-11ea-90b4-3ee54bc29d72.jpg)

**Comment by blackcrack** [comment]
jetzt sag mir du warum das bild nich geladen werden kann..
ehh  no let me show why the f.. d.. picture not be loaded \*bg*
hmmmmmmmm...

viel spaß beim austesten.. ich typer wieder mein /theme in die config.
have fun by testing out, i rewrite the theme again in my config.php

**Comment by mwalbeck** [comment]
My bad, I remembered wrong, it has been a while since I tried to use svg's directly from the theme, but I decided to drop it because of other reasons. I purely use svg's encoded in base64 embedded directly in the theme.

This is not a bug though.

Read this, and setup pretty urls properly and then it works with the original path you used:
https://docs.nextcloud.com/server/18/admin_manual/installation/source_installation.html#pretty-urls
